packages/react-widgets/src/models/CategoryModel.js

```diff
@@ -63,59 +55,30 @@ export const buildSqlQueryToGetCategories = ({
   column,
   operation,
   operationColumn,
-  filters,
-  alias = 'name'
+  filters
 }) => {
   const query = `
     WITH all_categories as (
       SELECT
-        ${column} as ${alias}
+        ${column} as category
```

Comment:
I think removing the alias breaks PieWidget. Now 'name' is used for both CategoryW and PieW, but we also have the external param for that.

Reply:
Done